components/server/ee/src/user/eligibility-service.ts

```diff
@@ -247,6 +247,27 @@ export class EligibilityService {
         return subscriptions.filter(s => eligblePlans.includes(s.planId!)).length > 0;
     }
 
+    /**
+     * Returns true if the user ought to land on a workspace cluster that provides more resources
+     * compared to the default case.
+     */
+    async userGetsMoreResources(user: User): Promise<boolean> {
+        if (!this.config.enablePayment) {
+            // when payment is disabled users can do everything
+            return true;
+        }
+
+        const subscriptions = await this.subscriptionService.getNotYetCancelledSubscriptions(user, new Date().toISOString());
+        const eligblePlans = [
```

Comment:
only PRO plans are eligible? or is it PRO and everything above?

Reply:
Indeed, for now this is only the unleashed/pro plans. We might have to revisit this depending on the plans of customers affected by this change.